services/utils/__init__.py

```diff
@@ -2,13 +2,20 @@
 import sys
 import os
 import traceback
+import pkg_resources
 from urllib.parse import urlencode
+from multidict import MultiDict
 from aiohttp import web
+from functools import wraps
 from typing import Dict
 import logging
 
-# The latest commit hash of the repository, if set as an environment variable.
+version = pkg_resources.require("metadata_service")[0].version
```

Comment:
the UI should not depend on the metadata_service

Reply:
This refers to `setup.py` and the global package name happens to be  `metadata_service`. We are not referring to the actual `./services/metadata_service` here.

Ideally we would like to display service version information on the UI, but alternatively we could leave it out as well. Ideas?